transportable-udfs-trino-plugin/src/main/java/com/linkedin/transport/trino/TransportUDFClassLoader.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2019 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.transport.trino;
+
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.util.List;
+
+
+public class TransportUDFClassLoader extends URLClassLoader {
```

Comment:
can you add more commentary about (a) why we need a separate classloader, (b) why we have to load some of the classes from the parent classloader first? This will be non-intuitive to future readers.

Reply:
The comments to explain why this ClassLoader is needed and some classes are loaded by the parent ClassLoader are added.